### PR TITLE
Translation fa

### PR DESCRIPTION
**Pull request by mabkenar**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.fa.yml [hunk 1]
```diff
@@ -255,82 +255,82 @@ howto:
...
-    #     fetching_errors: 'What can I do if an article encounters errors during fetching?'
-    #     all_docs: 'And so many other articles!'
-    # support:
-    #     title: 'Support'
-    #     description: 'If you need some help, we are here for you.'
-    #     github: 'On GitHub'
-    #     email: 'By email'
-    #     gitter: 'On Gitter'
+    page_title: 'Quickstart'
+     intro:
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fa.yml]: Indentation problem here, you should have 8 spaces.

Comment (mabkenar) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fa.yml]: Oh I see. Maybe when I was manually deleting comments to make translations, I should have deleted the extra space after the '#' character.
Is it only the red part above that needs to be fixed, or is it everywhere in the file?

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fa.yml]: Just this whole paragraph

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fa.yml]: The indentation problem is still here.
You should add 3 spaces to the whole quickstart section, except for the page_title

File: src/Wallabag/CoreBundle/Resources/translations/messages.fa.yml [hunk 2]
```diff
@@ -255,82 +255,82 @@ howto:
...
-    #     fetching_errors: 'What can I do if an article encounters errors during fetching?'
-    #     all_docs: 'And so many other articles!'
-    # support:
-    #     title: 'Support'
-    #     description: 'If you need some help, we are here for you.'
-    #     github: 'On GitHub'
-    #     email: 'By email'
-    #     gitter: 'On Gitter'
+    page_title: 'Quickstart'
+        intro:
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fa.yml]: My bad, it should have 4 spaces :confused: Sorry.

**Comment by j0k3r** [comment]
Ok we'll do in an other way.
Could you translate the developer part? I'll fix indentation after merging your PR :wink:

**Comment by j0k3r** [comment]
@mabkenar I've fixed indentation. Feel free to provide an other PR to translate missing translations.
Thanks !

**Pull request merged**